libraries/AC_WPNav/AC_WPNav.h

```diff
@@ -162,8 +161,8 @@ class AC_WPNav
     ///
 
     // get target yaw in centi-degrees (used for wp and spline navigation)
-    float get_yaw() const;
-    float get_yaw_rate_cds() const;
+    float get_yaw() const {return _pos_control.get_heading();}
```

Comment:
duplicate comment but surely we don't want to have the same method named different things in AC_WPNav and AC_PosControl.

nitpick: normally we put spaces between the curly brackets and the word before/after.  so it should be, "{ return _pos_control.get_heading(); }"

Reply:
Done